downloader/src/main/kotlin/WorkingTree.kt

```diff
@@ -81,9 +81,11 @@ abstract class WorkingTree(val workingDir: File, val vcsType: String) {
         val versionNames = filterVersionNames(version, listRemoteTags(), project)
         return when {
             versionNames.isEmpty() ->
-                throw IOException("No matching tag found for version '$version'.")
+                throw IOException("No matching tag found. Please tag the release revision using a tag name containing "
+                        + "the version '$version'.")
```

Comment:
Personally, I'd prefer: "No matching tag found for version '$version'. Please create a tag whose name contains the version."